app/views/athletes/index.html.erb

```diff
@@ -15,7 +15,11 @@
           <th class="text-center">Team URL</th>
           <th class="text-center">Rating</th>
           <th>More Info</th>
-          <th>Add</th>
+          <th>
+            <% if policy(@athlete).create? %>
```

Comment:
I am confused here? Because we aren't creating a new athlete? Maybe I don't understand the policy correctly.

Reply:
We are just calling the policy file, we are not adding any athletes